intg/src/main/java/org/apache/atlas/model/audit/EntityAuditEventV2.java

```diff
@@ -132,7 +132,8 @@ public static EntityAuditActionV2 fromString(String strValue) {
     private String              eventKey;
     private AtlasEntity         entity;
     private EntityAuditType     type;
-    private Map<String, Object> detail;
+    private List<Map<String, Object>> detail;
```

Comment:
We are changing the type of detail here, and this is used in multiple places in the product. Product use this in multiple places for nested query. Make sure this isn't breaking anything.